### PR TITLE
Zach

### PR DESCRIPTION
**Pull request by zwbce**
Updated movement.c to incorporate cliff sensors. Color detection to be added later. @ElyonsAnointed @jwfehr @mfborman

### CODE REVIEW

**Comment by ghost** [comment]
Looks good! +1

File: movement.c [hunk 1]
```diff
@@ -75,19 +76,54 @@ void moveForwardWithBumper(oi_t*sensor, int centimeters)
...
+		}
+		if(sensor->cliff_right)
+		{
+			moveBackward(sensor, reverseDistance);
+			sum -= reverseDistance * 10;
+			char str[] = "\nRIGHT CLIFF SENSOR ACTIVATED";
+			serial_puts(str);
+			break;
+		}
+		oi_set_wheels(fullSpeed, fullSpeed);
```
Comment (ghost) [inline movement.c:123]: This line should be removed. Already moving at fullSpeed.

Comment (mfborman) [inline movement.c:123]: I don't think this line is necessary

**Comment by mfborman** [comment]
+1

**Comment by jwfehr** [comment]
+1

**Pull request merged**